receiver/elasticsearchreceiver/scraper.go

```diff
@@ -56,6 +57,11 @@ var (
 	}
 )
 
+var es7_10 = func() *version.Version {
+	v, _ := version.NewVersion("7.10")
+	return v
+}
```

Comment:
```suggestion
}()
```

I believe this needs to be called.